### PR TITLE
Give users the ability to request mentorship

### LINKED ISSUE #26: As a new user, I would like the bot to tell me how to sign up for mentorship.
We want to make sure that our bot lets new users know, right away, that they can sign up for mentorship from Operation Code!

Requirements;
1. OcBot provides an additional button "I need a mentor!"
2. This button leads to a block of text describing the mentorship program.
3. In this block of text is a link to the Airtable form, allowing a user to enroll in the mentorship program.

### PR DESCRIPTION
**Pull request by tarebyte**
Closes #26 

This PR adds a new `Mentorship` button in the help menu that will (when finished) allow a user to go through the process of signing up for a mentor.

I'm wasn't entirely sure what would be the best description or the best link, so any resources would be much appreciated!

### CODE REVIEW

File: views/help_menu/mentorship_message.txt.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TODO TEXT GOES HERE
```
Comment (rickr) [inline views/help_menu/mentorship_message.txt.erb]: Lets add the following text here.

The Operation Code Mentorship program aims to pair you with an experienced developer in order to further your programming or career goals. When you sign up for our mentorship program you'll fill out a form with your experience and interests. You'll then be paired up with an available mentor that best meets those interests.

If you're interested in joining our mentorship program please fill out our sign up form here: http://op.co.de/mentee-enrollment.

Comment (tarebyte) [inline views/help_menu/mentorship_message.txt.erb]: Done in 8638f49!

**Comment by rickr** [comment]
Are there any other changes you wanted to get into this PR? The WIP tag is still on and I don't want to merge before you're ready.

**Comment by tarebyte** [comment]
> The WIP tag is still on and I don't want to merge before you're ready.

Nope just forgot to remove the tag, thanks @rickr!

**Pull request merged**